index.src.html

```diff
@@ -2530,6 +2548,11 @@
   [EnforceRange] unsigned long long duration;  // microseconds
   [EnforceRange] long long timestamp;          // microseconds
 };
+
+enum AlphaOption {
```

Comment:
Since this is used both by VideoFrame and VideoEncoderConfig, I would move this definition to its own section under configs (parallel to hardware acceleration) and add a few short <dfn> blurbs about the values.